plugin/webhook/webhook.go

```diff
@@ -34,13 +34,32 @@ const (
 	WebhookError Level = "ERROR"
 )
 
+// Issue object of issue
+type Issue struct {
+	ID          int    `json:"id"`
```

Comment:
In your particular case, do you need assignee_id, assignee_name, creator_id, creator_name?

Reply:
In fact, my case only needs the status of the issue